tests/integration/test_script_runner.py

```diff
@@ -120,12 +120,14 @@ class BackendDependencies(enum.Enum):
         "data_context_dir": "tests/integration/fixtures/no_datasources/great_expectations",
         "data_dir": "tests/test_sets/dataconnector_docs",
     },
-    {
-        "name": "rule_base_profiler_multi_batch_example",
-        "data_context_dir": "tests/integration/fixtures/yellow_tripdata_pandas_fixture/great_expectations",
-        "data_dir": "tests/test_sets/taxi_yellow_tripdata_samples/first_3_files",
-        "user_flow_script": "tests/integration/docusaurus/expectations/advanced/multi_batch_rule_based_profiler_example.py",
-    },
+    # Chetan - 20220125 - Commenting out RBP-specific test due to the feature and related doc undergoing significant changes as part of development.
+    #                     Both the doc and test will be rewritten after the feature is complete - only then should this be reenabled.
+    # {
+    #     "name": "rule_base_profiler_multi_batch_example",
+    #     "data_context_dir": "tests/integration/fixtures/yellow_tripdata_pandas_fixture/great_expectations",
+    #     "data_dir": "tests/test_sets/taxi_yellow_tripdata_samples/first_3_files",
+    #     "user_flow_script": "tests/integration/docusaurus/expectations/advanced/multi_batch_rule_based_profiler_example.py",
+    # },
```

Comment:
Per discussion with @alexsherstinsky and @Shinnnyshinshin, the doc here will drastically change as we promote the experimental RBP to production. As such, it makes sense to comment out the underlying test (which is directly used in the docs through Docusaurus embedding) until we get a better state.